NOTICE.md

```diff
@@ -11,6 +11,20 @@ The project maintains the following source code repositories:
 
  * https://github.com/eclipse/kiso
 
+## Contributors
+
+Thanks goes to these wonderful people (sorted by last name):
+
+ * Lars Beseke
+ * Michael Bierkandt
```

Comment:
I do not see Michael committing code in Kiso and signing MA-Vereinbarung, if you are talking about code contribution here. Of course, he should be in the list, if this is meant to be a general thanks giving list.

Reply:
Actually, he has signed the "MA-Vereinbarung" and the commit history shows his name. So, to be fair, I added him here, too.